cosmoz-tabbed-template-behavior.html

```diff
@@ -0,0 +1,90 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../cosmoz-templatize/cosmoz-templatize.html">
+
+<script>
+/**
+ `Cosmoz.TabbedTemplateBehavior`
+ */
+(function () {
+	'use strict';
+
+	window.Cosmoz = window.Cosmoz || {};
+
+	/**
+	* @polymerBehavior Cosmoz.TabbedTemplateBehavior
+	*/
+	Cosmoz.TabbedTemplateBehaviorImpl = {
+		properties: {
+
+			/**
+			 * If it is set to true the template will be stamped
+			 */
+			prerender: {
+				type: Boolean,
+				value: false,
+				observer: '_prerenderChanged'
+			}
+		},
+
+		attached() {
+			this._observer = Polymer.dom(this).observeNodes(this._onNodesChanged);
+		},
+
+		detached() {
+			Polymer.dom(this).unobserveNodes(this._observer);
+			this._observer = null;
+			this._templateInstance = null;
+		},
+
+		_onNodesChanged() {
+			if (this._userTemplate) {
+				return;
+			}
+			const template = this.queryEffectiveChildren('template');
+			if (!template) {
```

Comment:
This is not needed because `cosmoz-tab` should work with direct children and no template.

Reply:
no template warning removed in commit 569ec6d